python/ray/serve/scripts.py

```diff
@@ -729,6 +729,22 @@ def status(address: str, name: Optional[str]):
 def shutdown(address: str, yes: bool):
     warn_if_agent_address_set()
 
+    # check if the address is a valid Ray address
+    try:
+        # see what applications are deployed on the cluster
+        details = ServeSubmissionClient(address).get_serve_details()
+        apps = details.get("applications", {})
+        if len(apps) == 0:
+            cli_logger.warning(
+                f"Cannot shutdown Serve applications on {address} because there are no applications currently deployed."
```

Comment:
Hmm actually `serve shutdown` doesn't only pertain to the applications, it also shuts down the controller and proxies. I think it would be better to just modify the success log instead, so that it says something different than `applications will be deleted asynchronously` which can be confusing if there are no applications.